merkle/objhasher/objhasher.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
Did you mean to use 2016 or should we go with 2017?

Ditto for the other file.